tests/virt_autotest/vgpu.pm

```diff
@@ -0,0 +1,334 @@
+# SUSE's openQA tests
+# Copyright 2020 SUSE LLC
+# SPDX-License-Identifier: GPL-2.0-or-later
+#
+# Summary: A test to pass an NVIDIA vGPU to guest via libvirt. Only KVM hosts are supported.
+# Test environment: one AMPERE GPU card, such A10, A30 etc., with SR-IOV capability and/or MIG features in host machine;
+#                   A UEFI guest vm is defined in the host, and they are ssh accessible from host.
+# Test flow:
+#    - install the NVIDIA vGPU manager on host and reboot
+#    - create a GPU Instance and a Compute Instance if MIG feature is available
```

Comment:
Thank you for the work!

Just a reminder! As we talked earlier, on the 2 required milestones, we do full gpu test(for all you listed here). For other milestones, we do shorter test to only ensure that the vgpu manager is installable. Maybe the second item here can also be included in the short test. They are both preconditions for testing vgpu from virtualization regard. 

Please help make this easily achievable.

Reply:
> Just a reminder! As we talked earlier, on the 2 required milestones, we do full gpu test(for all you listed here). For other milestones, we do shorter test to only ensure that the vgpu manager is installable. Maybe the second item here can also be included in the short test. They are both preconditions for testing vgpu from virtualization regard.

Yes, the full vgpu passthough tests are included except licensing within vm guest. It does not take too more cost between a shorter test and a full test.

> Please help make this easily achievable.

What do you mean here?

Reply:
> > Just a reminder! As we talked earlier, on the 2 required milestones, we do full gpu test(for all you listed here). For other milestones, we do shorter test to only ensure that the vgpu manager is installable. Maybe the second item here can also be included in the short test. They are both preconditions for testing vgpu from virtualization regard.
> 
> Yes, the full vgpu passthough tests are included except licensing within vm guest. It does not take too more cost to run a full test than a shorter test.
> 
> > Please help make this easily achievable.
> 
> What do you mean here?

Differentiating short and full test, is not by the cost/effort, but by purpose. 

Given that vgpu driver version is changing, we decide that we will run short test for *every* milestone. We hope to see GREEN jobs if the short path is passing. For the required 2 milestones, we will run the full test.
To get the short and full tests differently, it maybe needed to have 2 testsuites for gpu. They will likely to reuse the vgpu.pm, but may need a switch to tell if it is short or full test and where to stop.

Reply:
oh, get it. Let me think it over how to implement it.

Reply:
introduced a new setting `VGPU_SHORT_TEST`